pkg/cluster/storageaccounts.go

```diff
@@ -70,10 +70,18 @@ func (m *manager) enableServiceEndpoints(ctx context.Context) error {
 	return nil
 }
 
+func (m *manager) isWorkerProfileAvailable() bool {
```

Comment:
There is no need to add additional complexity and overhead with a function to check the worker profile length, because we are only doing the length check in one place in this file.

Reply:
@darthhexx : updated as suggested